### PR TITLE
request to merge assignment 9 into develop branch

### PR DESCRIPTION
**Pull request by munendra-successive**
1.create repo and add Readme.md file

2.Install the lodash package using npm install lodash. and use any of its function

3.create a new folder named 'math' and create a file math.js. In math.js, define two functions:
a. add(a, b): takes two numbers as arguments and returns their sum.
b. subtract(a, b): takes two numbers as arguments and returns their difference.
create a new file index.js and just export those functions and then use those function in any of the file

### CODE REVIEW

File: assignment-9/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/*
+create a new folder named 'math' and create a file math.js. In math.js, define two functions:
+a. add(a, b): takes two numbers as arguments and returns their sum.
```
Comment (UrvashiChaudhary) [inline assignment-9/index.js:3]: Expected: create index.js file within math folder and export the functions from there. Then use it any of the file.

File: assignment-9/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "type": "module",
+  "license": "ISC",
+  "dependencies": {
+    "lodash": "^4.17.21"
```
Comment (UrvashiChaudhary) [inline assignment-9/package.json:13]: Please remove carat(^) symbol

**Review by UrvashiChaudhary — CHANGES_REQUESTED** [review]

File: assignment-9/index.js (hunk 1 above)
Comment (munendra-successive) [reply assignment-9/index.js]: Created index.js in math folder

File: assignment-9/package.json (hunk 2 above)
Comment (munendra-successive) [reply assignment-9/package.json]: carat symbol removed

**Review by UrvashiChaudhary — APPROVED** [review]

**Pull request merged**